### PR TITLE
Lat / Lng Removal

### PR DESCRIPTION
**Pull request by gokaysatir**
Change-Id: I43f9e744882c8d9158e07cc8f66a029c42f0a9fc

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by gokaysatir** [comment]
@timar i don't think this can pass CI, fyi. It'd be surprising.

File: browser/src/layer/SplitPanesContext.ts [hunk 1]
```diff
@@ -195,6 +195,60 @@ export class SplitPanesContext {
...
+
+			// There is vertical split, narrow down the initial view.
+			viewRectangles[0].pX1 = 0;
+			viewRectangles[0].pX2 = this._splitPos.x;
+		}
+
+		if (this._splitPos.y) {
+			const bottomLeftPane = viewRectangles[0].clone();
+			bottomLeftPane.pY1 = this._splitPos.y;
+
```
Comment (eszkadev) [inline browser/src/layer/SplitPanesContext.ts]: no `viewRectangles.push` ?

File: browser/src/layer/tile/CanvasSectionContainer.ts [hunk 2]
```diff
@@ -1667,6 +1670,9 @@ class CanvasSectionContainer {
 		this.canvas.style.width = cssWidth.toFixed(4) + 'px';
 		this.canvas.style.height = cssHeight.toFixed(4) + 'px';
 
+		app.canvasSize.pX = newWidth;
```
Comment (eszkadev) [inline browser/src/layer/tile/CanvasSectionContainer.ts:1673]: above we have some calculations we could already avoid by using SimplePoint created from newWidth / newHeight probably

File: browser/src/layer/tile/CellCursorSection.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+			drawingOrder: L.CSections.AutoFillMarker.drawingOrder,
+			zIndex: L.CSections.AutoFillMarker.zIndex,
+			interactable: true,
+			sectionProperties: {},
+		});
+
+		this.documentObject = true;
+
+		this.sectionProperties.viewId = viewId;
+        this.sectionProperties.cursorWeight = 2;
```
Comment (eszkadev) [inline browser/src/layer/tile/CellCursorSection.ts:32]: indent

File: browser/src/layer/SplitPanesContext.ts (hunk 1 above)
Comment (eszkadev) [reply browser/src/layer/SplitPanesContext.ts]: fixed in the next commit

File: browser/src/layer/tile/CanvasSectionContainer.ts (hunk 2 above)
Comment (gokaysatir) [reply browser/src/layer/tile/CanvasSectionContainer.ts:1673]: Those lines are for canvas resizing and carefully set for crisp images. Our simplePoint and simpleRectangle classes round the values. Canvas sizing uses floating numbers. I think they are good for now, unless we are really sure that it wouldn't break image quality. I also commented about this in geometry.ts file. Thanks for the comment.

**Comment by gokaysatir** [comment]
cell_cursor test was failing with 1px difference. I updated the range.

**Pull request merged**